snappy/snap_remote_repo.go

```diff
@@ -212,7 +214,18 @@ func (s *SnapUbuntuStoreRepository) Snap(name, channel string) (*RemoteSnap, err
 		return nil, err
 	}
 
-	return NewRemoteSnap(detailsData), nil
+	return NewRemoteSnap(detailsData, suggestedCurrency), nil
+}
+
+func getSuggestedCurrency(h *http.Header) string {
+	s := h.Get("X-Suggested-Currency")
+	if s == "" {
+		s = os.Getenv("U1_SEARCH_CURRENCY")
```

Comment:
I actually just copied this behaviour wholesale from the click scope without thinking about it too much. I'm tempted to just remove the environment variable from it altogether, unless we have a compelling use-case for it.